docs/_appendix/github-intro.md

```diff
@@ -56,9 +56,7 @@ Now that you have Git set up, you will need to fork the test repository. This
 will enable you to [submit][submit] your tests using a pull request (more on this
 [below][submit]).
 
-1.  In the browser, go the the GitHub page for the test repository:
-
-    Main W3C test repository: [github.com/w3c/web-platform-tests][main-repo]
+1.  In the browser, go the the GitHub page for the test repository: [github.com/w3c/web-platform-tests][main-repo]
```

Comment:
I think we could make this read even better:

`In the browser, go to [web-platform-tests on GitHub](link)`